beancount/loader.py

```diff
@@ -206,6 +206,7 @@ def wrapped(toplevel_filename, *args, **kw):
                 try:
                     result = pickle.load(file)
                 except Exception as exc:
+                    print("failed to load cache", exc)
```

Comment:
remove print

Reply:
Yes it't not ready for review yet. I use this to find why test failed in ci.